server.js

```diff
@@ -1,10 +1,192 @@
+  var client = new pg.Client(conString);
+
+  client.connect(function (err) {
+    if (err) throw err;
+    client.query('SELECT COUNT(DISTINCT Visitorpollid) "visitor_count" FROM visitorpoll', function(err, result) {
+      if (err) throw err;
+      client.end(function (err) {
+        if (err) throw err;
+      });
+      console.log({results: result.rows});
+      res.send(result.rows);
+    });
+  });
+});
+
+app.get('/location', function(req, res){
+  var conString = process.env.DATABASE_URL || null;
+  var client = new pg.Client(conString);
+
+  client.connect(function (err) {
+    if (err) throw err;
+    client.query('SELECT COUNT(DISTINCT visitoranswer.Visitorpollid) AS "count", ' +
+      '(CASE when question.Questionid = 7005 then answer.Answertext end) AS "country_answer", ' +
+      '(CASE when question.Questionid = 7004 then answer.Answertext end) AS "state_answer", ' +
+      'answer.country, answer.state, answer.lat, answer.lng, question.Questiontext, poll.Polltext ' +
+      'FROM visitoranswer INNER JOIN question ON visitoranswer.Questionid = question.QuestionID ' +
+      'INNER JOIN answer ON question.Questionid = answer.Questionid AND visitoranswer.Answerid = answer.Answerid ' +
+      'INNER JOIN poll ON poll.Pollid = question.Pollid ' +
+      'WHERE poll.Pollid = 7 AND question.Questionid = 7005 OR question.Questionid = 7004 ' +
+      'GROUP BY poll.polltext, question.Questiontext, question.Questionid, answer.Answertext, answer.country, answer.state, answer.lat, answer.lng;',
+      function(err, result) {
+        if (err) throw err;
+        client.end(function (err) {
+          if (err) throw err;
+        });
+        // console.log({results: result.rows});
+        res.send(result.rows);
+      });
+  });
+});
+
+app.get('/ages', function(req, res){
+  var conString = process.env.DATABASE_URL || null;
+  var client = new pg.Client(conString);
+
+  client.connect(function (err) {
+    if (err) throw err;
+    client.query('SELECT COUNT(DISTINCT visitoranswer.Visitorpollid) "Count", answer.Answertext, question.Questiontext, poll.Polltext ' +
+    	'FROM visitoranswer LEFT JOIN question ON visitoranswer.Questionid = question.QuestionID ' +
+      	'LEFT JOIN answer ON question.Questionid = answer.Questionid AND visitoranswer.Answerid = answer.Answerid ' +
+      	'INNER JOIN poll ON poll.Pollid = question.Pollid ' +
+      'WHERE poll.Pollid = 7 AND ' +
+    	 'question.Questionid = 7002 ' +
+      'GROUP BY poll.polltext, question.Questiontext, answer.Answertext ',
+      function(err, result) {
+        if (err) throw err;
+        client.end(function (err) {
+          if (err) throw err;
+        });
+        // console.log({results: result.rows});
+        res.send(result.rows);
+      });
+  });
+});
+
+app.get('/data/*', function(req, res){
+  var conString = process.env.DATABASE_URL || null;
+  var client = new pg.Client(conString);
+
+  var question;
+
+  switch(req.params[0]) {
```

Comment:
Rather than a switch, I would just create an object literal with key/value pairs. That will be less code, and can be easily retrieved by the key.